addons/mail/static/src/core/web/thread_service_patch.js

```diff
@@ -199,6 +215,23 @@ patch(ThreadService.prototype, "mail/core/web", {
             }
         }
     },
+    async loadMoreRecipients(thread) {
+        const recipients = await this.orm.call(thread.model, "message_get_followers", [
+            [thread.id],
+            Array.from(thread.recipients).at(-1).id,
+            100,
+            true,
```

Comment:
- No need to explicit `100`, it's default in python
- I think you can pass `kwargs` rather than positional argument. Would be more explicit than a lonely `true`.